statistics/compute_stats_zurich.py

```diff
@@ -0,0 +1,1330 @@
+    parser.add_argument('-exclude',
+                        metavar='<file>',
+                        required=False,
+                        help=
+                        "R|Config yaml file listing subjects to exclude from statistical analysis.\n"
+                        "Yaml file can be validated at this website: http://www.yamllint.com/.\n"
+                        "Below is an example yaml file:\n"
+                        + dedent(
+                                 """
+                                 - sub-1000032_T1w.nii.gz
+                                 - sub-1000498_T1w.nii.gz
+                                 """))
+    return parser
+
+
+def csv2dataFrame(filename):
+    """
+    Loads a .csv file and builds a pandas dataFrame of the data
+    Args:
+        filename (str): filename of the .csv file
+    Returns:
+        data (pd.dataFrame): pandas dataframe of the .csv file's data
+    """
+    #print(filename)
+    data = pd.read_csv(filename, encoding='utf-8')
+    return data
+
+
+def read_MSCC(path_results, exclude, df_participants, file_metric):
+    list_files_results = os.listdir(path_results)
+    list_files_results = [file for file in list_files_results if '_norm.csv' in file]  # Only take norm (include both)
+    subjects_list = np.unique([sub.split('_')[0] for sub in list_files_results])
+    print(len(subjects_list))
+
+    # TODO : check to simply add inside the participants.tsv 
+    columns = ['participant_id', 'level'] + METRICS + METRICS_NORM
+    df_morphometrics = pd.DataFrame(columns = columns) # todo add columns of metrics and
+    df_morphometrics['participant_id'] = subjects_list
+    for sub in subjects_list:
+        # Check if subject is in exlude list
+        files_subject = [file for file in list_files_results if sub in file]
+        df = csv2dataFrame(os.path.join(path_results, files_subject[0]))
+        max_level = df_participants.loc[df_participants['participant_id']==sub, 'maximum_stenosis'].to_list()[0]
+        all_compressed_levels = df_participants.loc[df_participants['participant_id']==sub, 'stenosis'].to_list()[0].split(', ')
+        idx_max = all_compressed_levels.index(max_level)
+        if idx_max not in df.index.to_list():
+            print(f'Maximum level of compression {max_level} is not in axial FOV, excluding {sub}')
+            df_morphometrics.drop(df_morphometrics.loc[df_morphometrics['participant_id']==sub].index)
+            exclude.append(sub)
+        else:
+            df_morphometrics.loc[df_morphometrics['participant_id']==sub, 'level'] = max_level
+            for metric in METRICS:
+                file = [file for file in files_subject if metric in file]#[0]
+                df = csv2dataFrame(os.path.join(path_results, file[0]))
+                # Fill list to create final df
+                column_norm = 'normalized_'+ metric + '_ratio'
+                column_no_norm = metric + '_ratio'
```

Comment:
TODO: Update column names to be compatible with changes proposed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4003#discussion_r1246007753